examples/carousel/js/carouselTabpanel.js

```diff
@@ -0,0 +1,44 @@
+/*
+*   File:   CarouselTabpanel.js
+*
+*   Desc:   Carousel Tab widget that implements ARIA Authoring Practices
+*/
+
+/*
+*   @constructor CarouselTabpanelpanel
+*
+*
+*/
+var CarouselTabpanel = function (domNode, carouselObj) {
+  this.domNode = domNode;
+  this.carousel = carouselObj;
+};
+
+CarouselTabpanel.prototype.init = function () {
+  this.domNode.addEventListener('focusin', this.handleFocusIn.bind(this));
+  this.domNode.addEventListener('focusout', this.handleFocusOut.bind(this));
+};
+
+CarouselTabpanel.prototype.hide = function () {
+  this.domNode.classList.remove('active');
+  this.domNode.removeAttribute('tabindex');
+};
+
+CarouselTabpanel.prototype.show = function () {
+  this.domNode.classList.add('active');
+  this.domNode.setAttribute('tabindex', '0');
```

Comment:
Do the tabpanels need to be focusable, since they all contain a focusable element? I'm not sure which is better in this case, it just struck me as perhaps confusing to have two tab stops that visually appear to be in the same place